### DISCUSSION TITLE
GeoFence Costs is missing total cost field

### DISCUSSION THREAD

**Discussion by horst789github** [teslamate-org/teslamate#1259] [Ideas]
i type my kWh cost but testmate only track to the loaded energy... but my charging provider charge more costs because of "charging losses" so maybe a field for total costs is better or automatic add ~10% to costs for charging losses

![grafik](https://user-images.githubusercontent.com/35397597/106056802-5ab20b00-60ef-11eb-9f65-209f6b6d7304.png)

**Comment by joshentu** [comment]
I am sure the cost is calculated by the actual energy "used", not the energy "added", so already have the charging loss considered
